www/js/services.js

```diff
@@ -102,6 +102,91 @@ angular.module('emission.services', [])
                "Email cancel reported, seems to be an error on android");
         });
     }
+
+    this.dataCollectionSetConfig = function(config) {
+      return new Promise(function(resolve, reject) {
+          resolve(window.cordova.plugins.BEMDataCollection.setConfig(config));
+      });
+    };
+
+    this.dataCollectionGetConfig = function() {
+      return new Promise(function(reslove, reject) {
+          reslove(window.cordova.plugins.BEMDataCollection.getConfig());
+      });
+    };
+    this.serverSyncSetConfig = function(config) {
+      return new Promise(function(resolve, reject) {
+          resolve(window.cordova.plugins.BEMServerSync.setConfig(config));
+      });
+    };
+
+    this.serverSyncGetConfig = function() {
+      return new Promise(function(reslove, reject) {
+          reslove(window.cordova.plugins.BEMServerSync.getConfig());
+      });
+    };
+
+    this.getAccuracyOptions = function() {
+      return new Promise(function(reslove, reject) {
+          reslove(window.cordova.plugins.BEMDataCollection.getAccuracyOptions());
+      });
+    };
+
+    this.getUserEmail = function() {
+      return new Promise(function(reslove, reject) {
+        window.cordova.plugins.BEMJWTAuth.getUserEmail(function(response) {
+            reslove(response);
+        }, function(error) {
+            reject(error);
+        });
+      });
+    };
+
+    this.getState = function() {
+      return new Promise(function(reslove, reject) {
+        window.cordova.plugins.BEMDataCollection.getState(function(response) {
+            reslove(response);
+        }, function(error) {
+            reject(error);
+        });
+      });
+    };
+
+    this.getSettings = function() {
+      return new Promise(function(reslove, reject) {
+        window.cordova.plugins.BEMConnectionSettings.getSettings(function(response) {
```

Comment:
e.g. ServerSync is only used in control

```
bash-3.2$ grep -r BEMServerSync www/js
www/js/main.js:        promiseList.push(window.cordova.plugins.BEMServerSync.getConfig());
www/js/main.js:            window.cordova.plugins.BEMServerSync.forceSync(function(result) {
www/js/main.js:        window.cordova.plugins.BEMServerSync.setConfig($scope.settings.sync.new_config)
```

Auth is used in one place other than control

```
bash-3.2$ grep -r BEMJWTAuth www/js
www/js/intro.js:    window.cordova.plugins.BEMJWTAuth.signIn(function(userEmail) {
www/js/main.js:            window.cordova.plugins.BEMJWTAuth.getJWT(function(result) {
www/js/main.js:        window.cordova.plugins.BEMJWTAuth.getUserEmail(function(result) {
www/js/main.js:            window.cordova.plugins.BEMJWTAuth.getJWT(function(result) {
```

Ditto for `ConnectionSettings`

```
bash-3.2$ grep -r BEMConnectionSettings www/js
www/js/intro.js:    window.cordova.plugins.BEMConnectionSettings.getSettings(function(settings) {
www/js/main.js:        window.cordova.plugins.BEMConnectionSettings.getSettings(function(result) {
www/js/services.js:        window.cordova.plugins.BEMConnectionSettings.getSettings(
```